nvm.sh

```diff
@@ -859,7 +859,6 @@ nvm_get_arch() {
   case "$NVM_UNAME" in
     *x86_64*) NVM_ARCH=x64 ;;
     *i*86*) NVM_ARCH=x86 ;;
-    *armv6l*) NVM_ARCH=arm-pi ;;
```

Comment:
Yes, I can add the logic to support until http://nodejs.org/dist/v0.10.28/. 

But I believe anything after `v0.10.28` will fail as there is no more `arm-pi` builds anymore in the node release e.g. http://nodejs.org/dist/v0.10.36/